docs/pages/versions/unversioned/sdk/stripe.md

```diff
@@ -9,7 +9,7 @@ import SnackInline from '~/components/plugins/SnackInline';
 
 Expo includes support for [`@stripe/stripe-react-native`](https://github.com/stripe/stripe-react-native), which allows you to build delightful payment experiences in your native Android and iOS apps using React Native & Expo. This library provides powerful and customizable UI screens and elements that can be used out-of-the-box to collect your users' payment details.
 
-If you're looking for a quick example, check out [this Snack](https://snack.expo.io/@charliecruzan/stripe-react-native-example?platform=mydevice)!
+If you're looking for a quick example, check out [this Snack](https://snack.expo.dev/@charliecruzan/stripe-react-native-example?platform=mydevice)!
```

Comment:
don't think this change is necessary since the redirects _do_ seem to work here, but no need to change it

Reply:
Yeah that's correct, Snacks referenced by their url work correctly when redirected. Ideally we should replace this Snack with either an inline version or by hosting the files statically from the docs.